pkg/gatewayserver/io/ws/lbslns/upstream.go

```diff
@@ -583,8 +583,7 @@ func (f *lbsLNS) HandleUp(ctx context.Context, raw []byte, ids *ttnpb.GatewayIde
 		updateSessionID(ctx, SessionIDFromXTime(jreq.UpInfo.XTime))
 		ct := recordTime(jreq.RefTime, jreq.UpInfo.XTime, jreq.UpInfo.GPSTime, jreq.UpInfo.RxTime)
 		if err := conn.HandleUp(up, ct); err != nil {
-			logger.WithError(err).Warn("Failed to handle upstream message")
-			return nil, err
+			log.FromContext(ctx).WithError(err).Warn("Invalid Join Request")
```

Comment:
I think we should've kept the original `Failed` messages, since they apply to buffer full too.